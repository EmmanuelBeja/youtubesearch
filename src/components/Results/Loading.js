import React from 'react'

const Loading = () => {
  return (
    <div className="text-center loading-container">
      <img src="/sync.svg" className="loading-img" alt="loading" />
      <br />
      <br />
      <p>Loading...</p>
    </div>
  )
}

export default Loading
