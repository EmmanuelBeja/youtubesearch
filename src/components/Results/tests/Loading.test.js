import React from 'react'
import { shallow } from 'enzyme'
import Loading from '../Loading'

describe('Loading', () => {
  const shallowWrapper = shallow(<Loading />)

  it('should render correctly', () => {
    expect(shallowWrapper).toMatchSnapshot()
    expect(shallowWrapper.find('.loading-container').length).toEqual(1)
    expect(shallowWrapper.find('.loading-img').length).toEqual(1)
  })
})
