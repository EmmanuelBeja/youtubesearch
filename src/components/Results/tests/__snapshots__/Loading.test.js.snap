// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render correctly 1`] = `
<div
  className="text-center loading-container"
>
  <img
    alt="loading"
    className="loading-img"
    src="/sync.svg"
  />
  <br />
  <br />
  <p>
    Loading...
  </p>
</div>
`;
